src/java/org/apache/cassandra/schema/SchemaConstants.java

```diff
@@ -92,6 +92,14 @@ public static boolean isReplicatedSystemKeyspace(String keyspaceName)
         return REPLICATED_SYSTEM_KEYSPACE_NAMES.contains(keyspaceName.toLowerCase());
     }
 
+    /**
+     * @return whether or not the keyspace is the schema keyspace
+     */
+    public static boolean isSchemaKeyspace(String keyspaceName)
+    {
+        return keyspaceName.toLowerCase().equals(SCHEMA_KEYSPACE_NAME);
```

Comment:
nit: `equalsIgnoreCase()` is less verbose and does not generate a transient lower case string